src/components/map/map-header.tsx

```diff
@@ -0,0 +1,85 @@
+import style from './map.module.css';
+
+import { ButtonLink } from '../button';
```

Comment:
I think on shared components it is good to use absolute path here
```suggestion
import { ButtonLink } from '@atb/components/button';
```